refactor/config_data/config.json

```diff
@@ -79,6 +79,18 @@
       "description": "Ratio between durability and current sea temperature (unit: days / °C)",
       "value": 210
   },
+  "infection_delay_time_EMB": {
```

Comment:
I believe it would be better to group treatment-related data into a singled object indexed by treatment type. See for example what was done for pheno_resistance (obviously the names can be changed to taste).